test/test_v2/test_common.c

```diff
@@ -381,12 +392,174 @@ void pmixt_post_finalize(pmix_proc_t *this_proc, test_params *params, validation
     exit(EXIT_SUCCESS);
 }
 
+/* This function is used to validate values returned from calls to Get against sidechannel validation data */
 void pmixt_validate_predefined(pmix_proc_t *myproc, const pmix_key_t key, pmix_value_t *value, validation_params *val_params)
 {
     pmix_status_t rc = PMIX_ERROR;
     size_t *size;
-    // To be developed.
-    ;
+
```

Comment:
I would add a check to make sure the `value->type` is correct for the `key` - to catch if the server is returning a value with the wrong type (when can happen if an RM is new and accidentally puts in the wrong type).

Reply:
Ok great, thanks Josh.

Reply:
My solution in the updated commit is to pass the type in as another parameter from the test, and then verify it matches the value->type, rather than hard code the additional checks for each type/key pair. Sound reasonable?